### PR TITLE
interface: document title

### PR DESCRIPTION
**Pull request by tylershuster**
This changes the name of the app across the board from "OS1" to "Landscape."

It also changes the document title as you navigate around the app, including surfacing notification counts.

### CODE REVIEW

**Comment by tylershuster** [comment]
Fixes https://github.com/urbit/landscape/issues/203

**Comment by tylershuster** [comment]
addresses https://github.com/urbit/landscape/issues/195

**Comment by tylershuster** [comment]
addresses https://github.com/urbit/landscape/issues/201

**Comment by matildepark** [comment]
If you pull out the eyre change as a separate PR we can deploy this to next-js.

**Comment by tylershuster** [comment]
Great, should I squash now?

**Comment by matildepark** [comment]
Yeah.

File: pkg/interface/src/views/apps/launch/app.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function LaunchApp(props) {
   return (
     <>
       <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/apps/launch/app.js:31]: Needs `defer={false}`

File: pkg/interface/src/views/apps/term/app.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default class TermApp extends Component {
     return (
       <>
         <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/apps/term/app.js:49]: Needs `defer={false}`

File: pkg/interface/src/views/landscape/components/GroupsPane.tsx [hunk 3]
```diff
@@ -133,28 +134,36 @@ export function GroupsPane(props: GroupsPaneProps) {
...
-              selected={appPath}
-              {...props}
-              baseUrl={baseUrl}
-            >
-              <UnjoinedResource
-                graphKeys={props.graphKeys}
-                notebooks={props.notebooks}
-                inbox={props.inbox}
+            <>
+              <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/landscape/components/GroupsPane.tsx:146]: Needs `defer={false}`

File: pkg/interface/src/views/landscape/components/GroupsPane.tsx [hunk 4]
```diff
@@ -186,20 +195,26 @@ export function GroupsPane(props: GroupsPaneProps) {
...
-                p='4'
-              >
-                <Box p="4"><Text fontSize="0" color='gray'>
-                  {description}
-                </Text></Box>
-              </Col>
-              {popovers(routeProps, baseUrl)}
-            </Skeleton>
+            <>
+              <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/landscape/components/GroupsPane.tsx:201]: Needs `defer={false}`

File: pkg/interface/src/views/landscape/components/Resource.tsx [hunk 5]
```diff
@@ -34,47 +34,58 @@ export function Resource(props: ResourceProps) {
...
+    }
+  }
   return (
-    <Switch>
-      <Route
-        path={relativePath("/settings")}
-        render={(routeProps) => {
-          return (
+    <>
+      <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/landscape/components/Resource.tsx:45]: Needs `defer={false}`

File: pkg/interface/src/views/landscape/index.tsx [hunk 6]
```diff
@@ -78,71 +77,76 @@ export default class Landscape extends Component<LandscapeProps, {}> {
...
-            const baseUrl = `/~landscape${groupPath}`;
-            const ws: Workspace = { type: 'group', group: groupPath };
-
-            return (
-              <GroupsPane workspace={ws} baseUrl={baseUrl} {...props} />
-            )
-          }}/>
-        <Route path="/~landscape/home"
+      <>
+        <Helmet>
```
Comment (matildepark) [inline pkg/interface/src/views/landscape/index.tsx:81]: Needs `defer={false}`

**Review by matildepark — COMMENTED** [review]
Careful — we want the number to update when we're out of the tab. Also seems like a lot of different places when the title doesn't change anywhere — why `landscape/index.tsx` *and* its child components?

**Comment by tylershuster** [comment]
As for why both the parent and children:

On the home page we want it to say `(1) Landscape`
But on child pages we want `(1) Urbit Community`

and so forth

**Comment by tylershuster** [comment]
Ok, deferments added and tested

File: pkg/interface/src/views/apps/launch/app.js (hunk 1 above)
Comment (matildepark) [reply pkg/interface/src/views/apps/launch/app.js:31]: What about this one?

Comment (tylershuster) [reply pkg/interface/src/views/apps/launch/app.js]: You're too quick

**Pull request merged**